docs/griptape-tools/official-tools/variation-image-generation-client.md

```diff
@@ -1,6 +1,8 @@
 # VariationImageGenerationEngine
 
-This tool allows LLMs to generate variations of an input image from a text prompt.
+This Tool allows LLMs to generate variations of an input image from a text prompt. The input image can be provided either by its filepath or by its [Task Memory](../../griptape-framework/tools/task-memory.md) reference. 
+
+## Referencing an image by filepath
```

Comment:
I think these headers have traditionally been more of a title rather than a sentence, right? Maybe just `File Path` and `Task Memory` for the other one?

Reply:
`File Path` and `Task Memory` alone don't really make sense as titles here, but I've updated the titles to be more titley.